ReactAndroid/src/main/java/com/facebook/react/views/recyclerview/RecyclerViewBackedScrollViewManager.java

```diff
@@ -81,6 +83,17 @@ public void scrollTo(
   }
 
   @Override
+  public void scrollToEnd(
+      RecyclerViewBackedScrollView scrollView,
+      ReactScrollViewCommandHelper.ScrollToEndCommandData data) {
+    FLog.w(
+      ReactConstants.TAG,
+      "scrollToEnd is not implemented for RecyclerViewBackedScrollView." +
+      "RecyclerViewBackedScrollView is deprecated and will be removed. " +
```

Comment:
cc @sahrens, just to make sure this is also the direction he's going in with ListView